includes/Admin/Admin_AJAX.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Admin\Admin_AJAX
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Admin;
+
+/**
+ * Class to handle the Admin AJAX requests.
+ *
+ * @since n.e.x.t
+ */
+class Admin_AJAX {
+
+	/**
+	 * Nonce key.
+	 *
+	 * @since n.e.x.t
+	 * @var string
+	 */
+	private $nonce_key;
+
+	/**
+	 * Sets the nonce key for verification.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $nonce_key Nonce key.
+	 */
+	public function __construct( $nonce_key ) {
+		$this->nonce_key = $nonce_key;
+	}
```

Comment:
Why do we need this constructor parameter and property? That seems unnecessary to me.

I think we can make whichever string we want to choose a `const` in this class.